core/src/components/toast/animations/ios.leave.ts

```diff
@@ -13,20 +13,23 @@ export const iosLeaveAnimation = (baseEl: HTMLElement, position: string): Animat
   const root = getElementRoot(baseEl);
   const wrapperEl = root.querySelector('.toast-wrapper') as HTMLElement;
 
-  const bottom = `calc(-10px - var(--ion-safe-area-bottom, 0px))`;
-  const top = `calc(10px + var(--ion-safe-area-top, 0px))`;
-
   wrapperAnimation.addElement(wrapperEl);
 
+  /**
+   * For top/bottom positions, we use to() instead of fromTo() so that if
+   * the positionAnchor prop was used, we don't need to recalculate the
+   * toast and/or anchor's position and duplicate work from the enter
+   * animation.
+   */
   switch (position) {
     case 'top':
-      wrapperAnimation.fromTo('transform', `translateY(${top})`, 'translateY(-100%)');
+      wrapperAnimation.to('transform', 'translateY(-100%)');
```

Comment:
Implicit Keyframes are somewhat buggy on WebKit, so we may want to avoid it for now: https://caniuse.com/?search=implicit%20keyframes

It's gotten better since iOS 14, but you can see in the demo below that the leaving animation is still broken.

https://github.com/ionic-team/ionic-framework/assets/2721089/e349ff3a-22b2-4db8-98a5-8a483be0869f